.envrc.sample

```diff
@@ -4,3 +4,4 @@ export REACT_APP_API_ENDPOINT='http://localhost:3000'
 export REACT_APP_CLIENT_ID=
 export REACT_APP_CLIENT_SECRET=
 export REACT_APP_ENCRYPTION_KEY=
+export REACT_APP_APPLE_REDIRECT_URI=
```

Comment:
I'm using NGROK locally to meet Apple's requirements for the redirect URI

- development: `https://storysprout.ngrok.app` (tunnels to `localhost:3006`)
- production: `https://storysprout.app`